src/ui/marker.ts

```diff
@@ -0,0 +1,100 @@
+import maplibregl, { MapMouseEvent } from 'maplibre-gl';
+
+import RadarMap from './map';
```

Comment:
What does this file need the map reference for, just for typing? Also looks like this is a circular dependency.

Reply:
yeah it's for typing so technically it's okay*. I can explicitly import the type instead

Reply:
refactor now lives in this pr: https://github.com/radarlabs/radar-sdk-js/pull/153